poc/test_opaque_ake.sage

```diff
@@ -72,20 +73,21 @@ def test_core_protocol_serialization():
     assert cred_request == deserialized_request
     assert length == len(serialized_request)
 
-    cred_response = core.create_credential_response(cred_request, pkS_enc, oprf_seed, record.envU, idU, record.masking_key)
+    cred_response = core.create_credential_response(cred_request, pkS_enc, oprf_seed, record.envU, idU, record.masking_key, info)
     serialized_response = cred_response.serialize()
     deserialized_response, length = deserialize_credential_response(
         config, serialized_response)
     assert cred_response == deserialized_response
     assert length == len(serialized_response)
 
-    recovered_skU, recovered_pkS, recovered_export_key = core.recover_credentials(pwdU, cred_metadata, cred_response)
+    recovered_skU, recovered_pkS, recovered_export_key = core.recover_credentials(pwdU, cred_metadata, cred_response, info)
 
     # Check that recovered credentials match the registered credentials
     assert export_key == recovered_export_key
 
 def test_registration_and_authentication():
     idU = _as_bytes("Username")
+    info = _as_bytes("Username")
```

Comment:
Why not pass `idU` directly?

Reply:
To test whether it solves the 'NoneType' problem

Reply:
So we'll revert this, yes?

Reply:
of course ! This PR is still in draft btw ;)